deepchem/molnet/load_function/molnet_loader.py

```diff
@@ -88,7 +88,8 @@ def __init__(self, featurizer: Union[dc.feat.Featurizer, str],
                splitter: Union[dc.splits.Splitter, str, None],
                transformer_generators: List[Union[TransformerGenerator, str]],
                tasks: List[str], data_dir: Optional[str],
-               save_dir: Optional[str], **kwargs):
+               save_dir: Optional[str],
+               precomputed_splits: Optional[bool], **kwargs):
```

Comment:
`precomputed_splits=False`, otherwise you will have to change this for all loaders.

Reply:
Doesn't ```Optional``` account for it?

Reply:
`Optional` is just for type check

Reply:
Can you add the new argument to the docstring?  Be sure to include a sentence, "If this is True, the splitter argument is ignored."  Likewise add a sentence to the description of the splitter argument: "If precomputed_splits is True, this is ignored."